assets/prefabs/Gooseberry.prefab

```diff
@@ -0,0 +1,10 @@
+{
+  "parent": "engine:iconItem",
+  "DisplayName": {
+    "name": "Gooseberry"
+  },
+  "Item": {
+    "icon": "SimpleFarming:SimpleFarming#gooseberry",
```

Comment:
Should be `SimpleFarming:Gooseberry` - your image is in a separate file not in an image atlas (that's what the `#` indicates - look for entry "gooseberry" in the texture atlas file "SimpleFarming")